aspnetcore/host-and-deploy/windows-service/samples/2.x/AspNetCoreService/SampleApp.csproj

```
@@ -0,0 +1,21 @@
+﻿<Project Sdk="Microsoft.NET.Sdk.Web">
+
+  <PropertyGroup>
+    <TargetFramework>netcoreapp2.2</TargetFramework>
+    <SelfContained>false</SelfContained>
+    <RuntimeIdentifier>win7-x64</RuntimeIdentifier>
+    <IsTransformWebConfigDisabled>true</IsTransformWebConfigDisabled>
+  </PropertyGroup>
+
+  <ItemGroup>
+    <PackageReference Include="Microsoft.AspNetCore.App" />
+    <PackageReference Include="Microsoft.AspNetCore.Hosting.WindowsServices" Version="2.2.0-preview3-35497" />
```

```suggestion
    <PackageReference Include="Microsoft.AspNetCore.Hosting.WindowsServices" Version="2.2.0" />
```

This packages will be 2.2.0 in the final RTM release...which is coming very soon.

---

I have to go with packages that are available at the time the topic is updated, but I've noted this for 2.2 RTM update cleanup work on https://github.com/aspnet/Docs/issues/8518.

---

Ok, that's fine.